Client.xcodeproj/project.pbxproj

```diff
@@ -12380,6 +12472,170 @@
 			defaultConfigurationName = Fennec;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		1B308A5F2714C9BD001D6200 /* XCRemoteSwiftPackageReference "rust-components-swift" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "/Users/skhamis/moz/rust-components-swift";
```

Comment:
Do we eventually have to change this repoURL to `https://github.com/mozilla/rust-components-swift`?

Reply:
This is a really good catch! I couldn't find where this was in the project as there is no reference to it so I wonder if it's just an artifact that didn't get cleaned up for some reason...will investigate